tike/coverage.py

```diff
@@ -67,9 +67,11 @@
 logger = logging.getLogger(__name__)
 
 
-def _coverage_interface(object_grid, object_corner, object_size,
-                        probe_grid, probe_size, theta, v, h,
-                        **kwargs):
+def _coverage_interface(
+    object_grid, object_corner, object_size,
+    probe_grid, probe_size, theta, v, h,
+    **kwargs
+):
     """Define an interface that all functions in this module match.
```

Comment:
Is this statement correct? Do you meant to say: "Define an interface that all reconstruction functions in this module match.".

Reply:
No. There are no reconstruction functions in the coverage module? This interface is for checking inputs of public functions. `tike.coverage.coverage` is the only function exported from this module.

Reply:
Do you expect to have multiple coverage functions?

Reply:
No. Good point.